files/templates/additional-scrape-configs.template.yaml

```diff
@@ -2228,4 +2228,33 @@
     replacement: {{ .ClusterType }}
   - target_label: provider
     replacement: {{ .Provider }}
+# prometheus
+- job_name: {{ .ClusterID }}-prometheus/prometheus-{{ .ClusterID }}/0
+  metrics_path: /metrics
+  scheme: http
+  kubernetes_sd_configs:
+  - role: endpoints
+  bearer_token_file: /var/run/secrets/kubernetes.io/serviceaccount/token
+  tls_config:
+    ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+    insecure_skip_verify: true
+  relabel_configs:
+  - source_labels: [__meta_kubernetes_pod_labelpresent_giantswarm_io_monitoring]
+    regex: (true)
+    action: keep
+  - source_labels: [__meta_kubernetes_service_labelpresent_operated_prometheus]
+    regex: (true)
+    action: keep    
```

Comment:
I don't get this. We already have the managed app config which should catch this.

Reply:
I think the managed app configs is checking it the annotation giantswarm.io is present, not the label. Let me check

Reply:
It seems we cannot change the service labels, only thse of the pods so we cannot manage it using the managed apps job at the moment

Reply:
ok I see. annotation and label confused me, sorry.